docs/draft/webview.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Dapp browser API usage
+version: 0.1.0
+status: Draft
+authors: 
+---
+
+# Dapp browser API usage
+    
+##  Table of Contents
+
+1. [Abstract](#abstract)
+2. [Definitions](#definitions)
+3. [Implementation](#implementation)
+4. [Usage](#usage)
+    * [Properties](#properties)
+        * [`isStatus`](#isStatus)
+        * [`status`](#status)
+    * [Methods](#methods)
+        * [`enable`](#enable)
+        * [`isConnected`](#isConnected)
+        * [`scanQRCode`](#scanQRCode)
+        * [`send`](#send)
+        * [`sendAsync`](#sendAsync)
+        * [`sendSync`](#sendSync)
+5. [Compatibility](#compatibility)
+6. [Changelog](#changelog)
+7. [Acknowledgements](#acknowledgements)
+8. [Copyright](#copyright)
+9. [Footnotes](#footnotes)
+
+## Abstract
+
+Status exposes an Ethereum Provider object (`window.ethereum`) to JS code running inside the browser. It also provides bidirectional communication between JS code and Status.
+
+## Definitions
+
+| Term       | Description                                                                         |
+|------------|-------------------------------------------------------------------------------------|
+| **Webview**   | React Native wrapper for a platform-specific browser core implementation.                                        |
+| **Ethereum Provider** | A JS object (`window.ethereum`) injected into each web page opened in the browser providing web3 support. |
+| **Bridge** | A set of facilities allow bidirectional communication between JS code and Status. |
+
+## Implementation
+Status uses a [forked version](https://github.com/status-im/react-native-webview) of [react-native-webview](https://github.com/react-native-community/react-native-webview)  to display web or dapps content. The fork provides an Android implementation of JS injection before page load. It is required in order to properly inject Ethereum Provider object.
+
+Status injects two JS scripts: 
+  - [provider.js](https://github.com/status-im/status-react/blob/develop/resources/js/provider.js): `window.ethereum` object
+  - [webview.js](https://github.com/status-im/status-react/blob/develop/resources/js/webview.js): override for `history.pushState` used internally
+
+Dapps running inside a browser communicate with Status Ethereum node by means of a *bridge* provided by react-native-webview library. It allows for bidirection communication between browser and Status. In order to do so, it injects a special `ReactNativeWebview` object into each page it loads. 
```

Comment:
"It allows.. It injects.. it provides.."
It is a bit hard to keep track of what we are talking about. I'd suggest to replace few "it"s to "bridge" to make it more clear.

Reply:
Thanks @vkjr, not quite clear indeed. Fixed.